src/components/Card/Card.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div class="card">
+    <a href="#">
+      <div>
+        <div class="image">
+          <img :src="ofert.thumbnail" alt="">
+        </div>
+      </div>
+    </a>
+    <div class="content-card">
+      <h5>
+        <a href="#">{{ofert.title | capitalize}}</a>
+      </h5>
+      <p>
+        {{ofert.location}}
+        <span class="featured-badge">Full time</span>
+      </p>
+      <p>{{ofert.desc}}</p>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    ofert: {
+      type: Object,
+      default () {
+        return {
+          title: '',
+          desc: '',
+          location: '',
+          thumbnail: ''
+        }
+      }
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+.content-card {
+  padding: 15px 15px 0 15px;
+  text-align: center;
```

Comment:
The text should be justified, and apply only to the paragraph, not all the elements of the Card. You can use [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) for the layout of the card!